mopidy_musicbox_webclient/static/js/functionsvars.js

```diff
@@ -176,185 +176,156 @@ function albumTracksToTable (pl, target, uri) {
     $(target).attr('data', uri)
 }
 
-function renderSongLi (song, liID, uri) {
-    var name
-    if (!song.name || song.name === '') {
-        name = uri.split('/')
-        name = decodeURI(name[name.length - 1])
+function renderSongLi (song, liID, uri, tlid, renderAlbumInfo) {
+    var name, iconClass
+    var tlidString = ''
+    var tlidParameter = ''
+    var onClick = ''
+    // Determine if the song line item will be rendered as part of an album.
+    if (!song.album || !song.album.name) {
+        iconClass = getMediaClass(song.uri)
     } else {
-        name = song.name
+        iconClass = 'trackname'
     }
-    songLi = '<li class="song albumli" id="' + liID + '">' +
-             '<a href="#" class="moreBtn" onclick="return popupTracks(event, \'' + uri + '\',\'' + song.uri + '\');">' +
+    // Play by tlid if available.
+    if (tlid) {
+        tlidString = '" tlid="' + tlid
+        tlidParameter = '\',\'' + tlid
+        onClick = 'return playTrackQueueByTlid(\'' + song.uri + '\',\'' + tlid + '\');'
```

Comment:
Am I being dense or can't you use double quotes for the outer quotes here and save the less readable escaping?